person.rb

```diff
@@ -1,16 +1,18 @@
 require './nameable'
 require './capitalize_decorator'
 require './trimmer_decorator'
+require './rental'
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [ ] A person has many rentals :+1:  Can we now include a method for adding rentals in this class? This method should be present.

Reply:
That is line 15 for, can you check

Reply:
@Flunavgt `@rentals = []` initialization? . Kindly include a method for adding rental please.

`hint`:

```
def add_rental(book, date)
 ..
end
```

Kindly note that the person itself should be sent as  a parameter to create the new rental achieving the has_many association

Reply:
I'm sorry to insist, but if you are able to run the test file you will notice that a person can record many rentals 
![Screen Shot 2022-11-02 at 19 24 48](https://user-images.githubusercontent.com/99816838/199632060-4fcd0a4a-5a0d-409b-a57e-5a0453d38376.png)

Reply:
Hi @Flunavgt, thanks for further explaining through the zoom call. I didn't pay attention that much. The previous reviewer will equally look into that.

Reply:
Once more, I think it's much better to have a single method that does that instead of creating instances of person, book, and rental.


In your Person class, a method like the one below plays the same role.

def add_rental(book, date)
  Rental.new(date, book, self)
end